package.json

```
@@ -59,14 +59,14 @@
         "@fastify/cors": "^9.0.1",
         "@fastify/swagger": "^8.15.0",
         "@fastify/swagger-ui": "^4.1.0",
-        "@kynesyslabs/demosdk": "^2.5.13",
+        "@kynesyslabs/demosdk": "^2.7.9",
         "@metaplex-foundation/js": "^0.20.1",
         "@modelcontextprotocol/sdk": "^1.13.3",
         "@noble/ed25519": "^3.0.0",
         "@noble/hashes": "^2.0.1",
         "@octokit/core": "^6.1.5",
```

Biome (2.1.2)

[error] The key @octokit/core was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)